tests/system/projects/test_project.py

```
@@ -1967,6 +1963,78 @@ def test_notifications_on_workflow_start_and_completion(self):
             len(notifications) == 2
         ), f"Expected 2 notifications, got {len(notifications)}"
 
+    @pytest.mark.parametrize("kind", ["nuclio", "job", "mpijob"])
+    def test_deploy_function_with_mounted_project_secret(self, kind):
```

I wonder, how "slow" this test is? covering all parameterized. if it fails fast, then should be good, right?

---

@liranbg yes, it's fast, cause it doesn't really deploy function and fails almost immediately after api gets the request